build.gradle

```diff
@@ -161,19 +161,18 @@ compileJava {
 dependencies {
     compile group: 'cobra', name: 'cobra', version:'0.98.4-pcgen'
     compile group: 'commons-io', name: 'commons-io', version:'2.6'
-    compile group: 'org.springframework', name: 'spring-core', version:'5.0.3.RELEASE'
-    compile group: 'org.springframework', name: 'spring-beans', version:'5.0.3.RELEASE'
-    compile group: 'org.springframework', name: 'spring-web', version:'5.0.3.RELEASE'
+    compile group: 'org.springframework', name: 'spring-core', version:'5.0.5.RELEASE'
+    compile group: 'org.springframework', name: 'spring-beans', version:'5.0.5.RELEASE'
```

Comment:
compile group: 'org.springframework', name: 'spring-web', version:'5.0.5.RELEASE'

Reply:
I've added it directly, lets see if this passes